V2/time_tracker/time_entries/_infrastructure/_data_persistence/_time_entries_dao.py

```diff
@@ -44,9 +44,14 @@ def create(self, time_entry_data: domain.TimeEntry) -> domain.TimeEntry:
         except sqlalchemy.exc.SQLAlchemyError:
             return None
 
-    def __create_time_entry_dto(self, time_entry: dict) -> domain.TimeEntry:
-        time_entry = {key: time_entry.get(key) for key in self.time_entry_key}
-        return domain.TimeEntry(**time_entry)
+    def update(self, time_entry_id: int, time_entry_data: dict) -> domain.TimeEntry:
+
+        query = self.time_entry.update().where(self.time_entry.c.id == time_entry_id).values(time_entry_data)
+        self.db.get_session().execute(query)
+        query_deleted_time_entry = sqlalchemy.sql.select(self.time_entry).where(self.time_entry.c.id == time_entry_id)
```

Comment:
Maybe the name of this variable is not accurate to the update method, please could change the name of the variable.